src/driver/main.cpp

```diff
@@ -505,8 +505,10 @@ struct roctx : command<roctx>
 struct op : command<op>
 {
     bool show_ops = false;
+    std::string op_name{};
     void parse(argument_parser& ap)
     {
+        ap(op_name, {}, ap.metavar("<operator name>"));
```

Comment:
Looks like this time it works fine. I would suggest change the "operator name" to "migraphx operator name" or "migraphx op name" to explicitly mention it is not an ONNX op name.